src/Core/Math/Sphere.js

```diff
@@ -0,0 +1,54 @@
+import THREE from 'THREE';
+import defaultValue from 'Core/defaultValue';
+
+
+function Sphere(center,radius) {
+
+	this.center = defaultValue(center,new THREE.Vector3());
+	this.radius = defaultValue(radius,1.0);
```

Comment:
What do you think of:
```js
this.center = center || new THREE.Vector3();
this.radius = radius || 1.0;
```
It's not exactly the same thing that `defaultValue` (which test against `undefined`) but it looks more Javascript-y.